src/ui/SearchInterface/SearchInterface.ts

```diff
@@ -484,7 +485,11 @@ export class SearchInterface extends RootComponent implements IComponentBindings
      *
      * @availablesince [March 2020 Release (v2.8521)](https://docs.coveo.com/en/3203/)
      */
-    enableScrollRestoration: ComponentOptions.buildBooleanOption({ defaultValue: false })
+    enableScrollRestoration: ComponentOptions.buildBooleanOption({ defaultValue: false }),
+    /**
+     * Specifies whether modal components should be child nodes of the search interface in the DOM.
+     */
+    attachModalsToSearchInterface: ComponentOptions.buildBooleanOption({ defaultValue: false })
```

Comment:
Why not move the option we had before here, but globally ?

So instead of "attach to search interface or `something` (and we don't know what something is)", we could choose the element itself. 

So something like `modalContainer: ComponentOptions.buildSelectorOption({ defaultFunction: element => element.ownerDocument.body })`